pandas/core/arrays/integer.py

```diff
@@ -85,6 +85,34 @@ def construct_array_type(cls):
         """
         return IntegerArray
 
+    def __from_arrow__(self, array):
+        """Construct IntegerArray from passed pyarrow Array"""
+        import pyarrow
+
+        if isinstance(array, pyarrow.Array):
```

Comment:
When is this False?

Reply:
The passed pyarrow values can be either a pyarrow.Array or pyarrow.ChunkedArray. Added a comment for this